Gemfile

```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
 gemspec
+gem "test-kitchen"
```

Comment:
@StylusEaterChef What requires us to add the test-kitchen gem?

Reply:
Good question. Not sure how that go in there, removed it. Thanks.